test/simulator/main/org/apache/cassandra/simulator/systems/InterceptorOfGlobalMethods.java

```diff
@@ -383,12 +404,12 @@ public static CountDownLatch newCountDownLatch(int count)
 
         public static Semaphore newSemaphore(int count)
         {
-            return new Standard(count, false);
+            return methods.newSemaphore(count);
         }
 
         public static Semaphore newFairSemaphore(int count)
         {
-            return new Standard(count, true);
+            return methods.newSemaphore(count);
```

Comment:
```suggestion
            return methods.newFairSemaphore(count);
```

this is doing `false`

Reply:
Well spotted, thanks!